source/change-streams/tests/unified/change-streams.yml

```diff
@@ -0,0 +1,74 @@
+description: "change-streams"
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: []
+      ignoreCommandMonitoringEvents: []
```

Comment:
`observeEvents` and `ignoreCommandMonitoringEvents` are both optional. If you're not specifying any events here, I suggest just omitting the fields entirely. In fact, empty arrays should fail to validate with the schema as it [requires `minItems: 1`](https://github.com/mongodb/specifications/blob/c6645365fb68e9427f8599196a790819942f4e64/source/unified-test-format/schema-1.1.json#L79) for both fields.

I realize the `client` [entity](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#entity) definition doesn't explicitly say these default to empty arrays. Rather, it says:

> Optional array of one or more strings.

If you think that's unclear, we can amend the spec language to explicitly say they default to an empty array. I was previously just relying on the schema validation itself to enforce this, and prevent folks from redundantly specifying an empty array.

Reply:
Thank you,
> Optional array of one or more strings.
is clear. The reason I violated the schema is that I either had not noticed this phrase in the unified format, or had not paid attention. This said, I agree that the JSON validation agains the schema should be made automatic.

For now I validated the compliance of `change-streams.json` to `unified-test-format/schema-1.1.json` with [justify](https://github.com/leadpony/justify#command-line-interface).

Reply:
@stIncMale pointed out that we do have some tests which assert server behavior and there are some drivers that model change stream documents as native types (e.g. structs/classes), so having a test might benefit at least a subset of drivers. It's also a relatively simple test to add so I'm fine having it.